sigs.yml

```diff
@@ -679,7 +679,7 @@
       - name: 'Austin Parker'
         github: austinlparker
   - name: Contributor Experience
-    meeting: Every Monday alternating between 9:00 and 17:00 UTC
+    meeting: Monday alternating between 10:00 PT and 17:00 UTC+8
```

Comment:
@open-telemetry/sig-contributor-experience-approvers please take a look if I did the math correctly